packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js

```diff
@@ -5,45 +5,66 @@
     For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 */
 import { LightningElement, wire, track, api } from 'lwc';
-import { productsByQuery } from 'commerce/data';
+import QUERY from './gqlTemplate';
+import { useQuery } from '@lwce/apollo-client';
+import '../api/client';
 
-//
-// Displays search results
-//
 export default class ProductSearchResults extends LightningElement {
     @track state = {};
     @track products = [];
     @track refinementgroups = [];
     sortingOptions = [];
-    @api query = '';
     @track loading = true;
     @track refinementBar = 'refinement-bar col-md-3 d-none d-lg-block';
     @track showRefinementBar = true;
-    sortRule;
     selectedRefinements = {};
 
-    // The wire adaptor to search for products when the query, sort rule or selected refinements change.
-    @wire(productsByQuery, {
-        query: '$query',
-        sortRule: '$sortRule',
-        selectedRefinements: '$selectedRefinements',
+    displayQuery = '';
+
+    variables = {
+        query: '',
+        filters: [],
+    };
+    sortRuleValue = '';
+
+    @api set query(val) {
+        this.displayQuery = val;
+        this.variables = { ...this.variables, query: encodeURIComponent(val) };
+    }
+
+    get query() {
+        return this.variables.query;
+    }
+
+    @api set sortRule(val) {
+        this.sortRuleValue = val;
```

Comment:
Should we use this._sortRule here and remove sortRuleValue as Jason C. mentioned above.
Same form get sortRule below.

Reply:
sortRuleValue is used in another function and unlike the query it will not be altered (encoding)